salt/metalk8s/saltmaster/files/salt-master-pod.yaml.j2

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: saltmaster
+  namespace: kube-system
+  labels:
+    app: saltmaster
+    app.kubernetes.io/name: saltmaster
+    app.kubernetes.io/version: '{{ saltmaster_version }}'
+    app.kubernetes.io/component: salt
+    heritage: metalk8s
+    app.kubernertes.io/part-of: metalk8s
+    app.kubernetes.io/managed-by: salt
+spec:
+  hostNetwork: true
+  priorityClassName: system-cluster-critical
+  securityContext:
+    runAsUser: 0
+    fsGroup: 0
+    readOnlyRootFilesystem: true
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+        - ALL
+  containers:
+    - name: saltmaster
+      image: docker.io/library/{{saltmaster_image}}:{{saltmaster_version}}
+      imagePullPolicy: Never
+      ports:
+        - name: master
+          containerPort: 4505
+          protocol: TCP
+        - name: api
```

Comment:
What does this `api` refer to? That's not `salt-api`, right.

Reply:
then we need to change the comment in the Dockerfile :p 
```
# salt-master, salt-api
EXPOSE 4505 4506
```